### PR TITLE
Don't mark PRs with "WIP" as "ready for review"

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

**Review by chalasr — APPROVED** [review]
Thanks!

**Review by jderusse — APPROVED** [review]
LGTM. Just few comments.

File: src/Service/WipParser.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class WipParser
+{
+    /**
+     * Returns true if the title has "[WIP]" in it.
+     */
+    public static function parseTitle(string $title): bool
```
[the remark this answers is not in the document]
Comment (Nyholm) [reply src/Service/WipParser.php:13]: Thank you

File: src/Service/WipParser.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+namespace App\Service;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class WipParser
+{
+    /**
+     * Returns true if the title has "[WIP]" in it.
```
[the remark this answers is not in the document]
Comment (Nyholm) [reply src/Service/WipParser.php]: Thank you

**Review by wouterj — APPROVED** [review]

**Comment by weaverryan** [comment]
Thank you Tobias!

**Pull request merged**